src/csharp/Microsoft.Spark/SparkContext.cs

```diff
@@ -386,5 +393,13 @@ private JvmObjectReference WrapAsJavaRDD(JvmObjectReference rdd)
                 "toJavaRDD",
                 rdd);
         }
+
+
+        /// <summary>
+        /// Returns a Version object that represents the version of Spark on which this application is running
+        /// </summary>
+        /// <returns>A Version object that represents the version of Spark on which this application is running</returns>
+
+        public Version Version() => _version;
```

Comment:
Let's return `string`.  Also you can call the SparkContext's `version` method like the following.
```suggestion
        public string Version() => (string)_jvmObject.Invoke("version");
```

Reply:
adjusted.